.travis.yml

```diff
@@ -1,86 +1,67 @@
 language: php
 
-sudo: false
-
-addons:
-  firefox: "47.0.1"
-  postgresql: "9.6"
+os: linux
+dist: xenial
 
 services:
-  mysql
+  - mysql
+  - postgresql
+  - docker
 
 cache:
   directories:
     - $HOME/.composer/cache
     - $HOME/.npm
+    - $HOME/.nvm
+
+jobs:
+  include:
+    - php: 7.4
+      env: MOODLE_BRANCH=master            DB=pgsql
+    - php: 7.4
+      env: MOODLE_BRANCH=master            DB=mysqli
+    - php: 7.4
+      env: MOODLE_BRANCH=MOODLE_311_STABLE DB=pgsql
+    - php: 7.4
+      env: MOODLE_BRANCH=MOODLE_311_STABLE DB=mysqli
+    - php: 7.4
+      env: MOODLE_BRANCH=MOODLE_310_STABLE DB=pgsql
+    - php: 7.4
+      env: MOODLE_BRANCH=MOODLE_39_STABLE  DB=mysqli
+    - php: 7.4
+      env: MOODLE_BRANCH=MOODLE_38_STABLE  DB=pgsql
+
+    - php: 7.3
+      env: MOODLE_BRANCH=MOODLE_37_STABLE  DB=mysqli
+
+    - php: 7.2
```

Comment:
Now that https://tracker.moodle.org/browse/MDL-69308 I guess it will be better to update the minimum PHP versions for master and 3.11.

Reply:
Done, I've moved all 311 and master tests from php72 to php73!